### DISCUSSION TITLE
Increase timeout for scalafmt

### DISCUSSION THREAD

**Discussion by ayoub-benali** [scalameta/nvim-metals#700] [Q&A] [answered]
Hello,
when I run the scalafmt command within neovim I often get a timeout message from metals. 
It is possible to increase this via a settings ?

**Comment by ckipp01** [comment, marked as answer]
Hey @ayoub-benali are you talking about the `vim.lsp.buf.format`? That's how formatting is triggered in neovim and if so it should just be in your config. You can add a `timeout_ms` to the `opts` table that you can pass in as a parameter.

**Reply by ayoub-benali** [reply]
Do you mean something like this ?
`keymap.set("n", "<leader>f", function() vim.lsp.buf.format({timeout_ms = 2000}) end, { desc = "LSP: Format"})`

**Reply by ckipp01** [reply]
yes, exactly.
